titus_isolate/numa/utils.py

```diff
@@ -0,0 +1,19 @@
+from titus_isolate import log
+
+
+def enable_numa_balancing():
+    log.info("Enabling NUMA balancing.")
+    _set_numa_balancing(1)
+
+
+def disable_numa_balancing():
+    log.info("Disabling NUMA balancing.")
+    _set_numa_balancing(0)
+
+
+def _set_numa_balancing(state: int):
+    if state != 0 and state != 1:
+        raise ValueError("numa_balancing may be set to either 0 or 1, received: '{}'".format(state))
+
+    with open('/proc/sys/kernel/numa_balancing', 'w') as f:
```

Comment:
I think it's also worth reading it, and checking if the value is set already. I don't know if the action of (re)setting this has any kernelspace side effects.

Reply:
I don't know either.  I did check that it always exits with a `0` exit code, even on repeat settings.  I can do the read though.